Tensile/KernelWriterAssembly.py

```diff
@@ -570,7 +570,12 @@ def getCompileArgs(self, sourceFileName, objectFileName, *moreArgs):
     if archHasV3:
       rv += ['-mno-code-object-v3' if globalParameters["CodeObjectVersion"] == "V2" else '-mcode-object-v3']
 
-    rv += ['-mcpu=gfx' + ''.join(map(str,isa))]
+    rv += ['-mcpu=gfx' + ''.join(map(str,isa)), '-mno-xnack']
+    if isa == (9,0,6):
+      rv += ['-mno-sram-ecc']
```

Comment:
is this wrong?

Reply:
> is this wrong?

No, this is for our assembly kernels which follow the current official documentation i.e. for gfx906, sram-ecc defaults to OFF.